x/auth/ante.go

```diff
@@ -69,7 +84,9 @@ func NewAnteHandler(ak AccountKeeper, fck FeeCollectionKeeper) sdk.AnteHandler {
 			return newCtx, err.Result(), true
 		}
 
-		if res := ValidateMemo(newCtx.GasMeter(), stdTx, params); !res.IsOK() {
+		newCtx.GasMeter().ConsumeGas(params.TxSizeCostPerByte*sdk.Gas(len(newCtx.TxBytes())), "txSize")
```

Comment:
Is there any potential overflow here? I expect the Tendermint-side size limits will be enough to ensure otherwise, but maybe we should check it anyways.

Reply:
hmmm I doubt we'll ever exceed `1<<64 - 1` for tx size, but perhaps it is indeed safer we add this check. I'm thinking `ConsumeGas` should check this and panic on overflow. Thoughts?

Reply:
That sounds reasonable to me.

Reply:
Ahh, but I recall I already added this check to `ConsumeGas`. Nothing to do here.